tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/debugger_container_test.ts

```diff
@@ -890,136 +894,107 @@ describe('Debugger Container', () => {
+      expect(filePathElements.length).toBe(3);
+      expect(filePathElements[0].nativeElement.innerText).toBe(
         stackFrame0[1].slice(stackFrame0[1].lastIndexOf('/') + 1)
       );
-      expect(filePathElements[0].nativeElement.title).toEqual(stackFrame0[1]);
-      expect(filePathElements[1].nativeElement.innerText).toEqual(
+      expect(filePathElements[0].nativeElement.title).toBe(stackFrame0[1]);
+      expect(filePathElements[1].nativeElement.innerText).toBe(
         stackFrame1[1].slice(stackFrame1[1].lastIndexOf('/') + 1)
       );
       expect(filePathElements[1].nativeElement.title).toEqual(stackFrame1[1]);
-      expect(filePathElements[2].nativeElement.innerText).toEqual(
+      expect(filePathElements[2].nativeElement.innerText).toBe(
         stackFrame2[1].slice(stackFrame2[1].lastIndexOf('/') + 1)
       );
-      expect(filePathElements[2].nativeElement.title).toEqual(stackFrame2[1]);
+      expect(filePathElements[2].nativeElement.title).toBe(stackFrame2[1]);
 
       const linenoElements = fixture.debugElement.queryAll(
         By.css('.stack-frame-lineno')
       );
-      expect(linenoElements.length).toEqual(3);
-      expect(linenoElements[0].nativeElement.innerText).toEqual(
+      expect(linenoElements.length).toBe(3);
+      expect(linenoElements[0].nativeElement.innerText).toBe(
         `Line ${stackFrame0[2]}`
       );
-      expect(linenoElements[1].nativeElement.innerText).toEqual(
+      expect(linenoElements[1].nativeElement.innerText).toBe(
         `Line ${stackFrame1[2]}`
       );
-      expect(linenoElements[2].nativeElement.innerText).toEqual(
+      expect(linenoElements[2].nativeElement.innerText).toBe(
         `Line ${stackFrame2[2]}`
       );
 
       const functionElements = fixture.debugElement.queryAll(
         By.css('.stack-frame-function')
       );
-      expect(functionElements.length).toEqual(3);
-      expect(functionElements[0].nativeElement.innerText).toEqual(
-        stackFrame0[3]
+      expect(functionElements.length).toBe(3);
+      expect(functionElements[0].nativeElement.innerText).toBe(stackFrame0[3]);
+      expect(functionElements[1].nativeElement.innerText).toBe(stackFrame1[3]);
+      expect(functionElements[2].nativeElement.innerText).toBe(stackFrame2[3]);
+
+      // Check the focused stack frame has been highlighted by CSS class.
+      const focusedElements = fixture.debugElement.queryAll(
+        By.css('.focused-stack-frame')
       );
-      expect(functionElements[1].nativeElement.innerText).toEqual(
-        stackFrame1[3]
+      expect(focusedElements.length).toBe(1);
+      const focusedFilePathElement = focusedElements[0].query(
+        By.css('.stack-frame-file-path')
       );
-      expect(functionElements[2].nativeElement.innerText).toEqual(
-        stackFrame2[3]
+      expect(focusedFilePathElement.nativeElement.innerText).toBe(
+        stackFrame1[1].slice(stackFrame1[1].lastIndexOf('/') + 1)
```

Comment:
Since this is a test, I'll opt not to add a helper function now. It's my preference to keep test logic explicit even when it's sometimes a little more verbose.